src/predicates.cpp

```diff
@@ -65,27 +65,37 @@ class TouchesPredicate {
 };
 
 void init_predicates(py::module& m) {
+    py::options options;
+    options.disable_function_signatures();
+
     m.def("intersects",
           py::vectorize(Predicate(s2geog::s2_intersects)),
           py::arg("a"),
           py::arg("b"),
-          R"pbdoc(
+          R"pbdoc(intersects(a, b)
+
         Returns True if A and B share any portion of space.
 
         Intersects implies that overlaps, touches and within are True.
 
         Parameters
         ----------
         a, b : :py:class:`Geography` or array_like
-            Geography object(s)
+            Geography object(s).
+
+        Returns
+        -------
+        bool or array
+            Results of the ``intersects`` test.
```

Comment:
Sidenote, and this is totally fine since you already added it! But personally I don't find it needed to strictly always have an _explanation_ of the result. It's good to know it returns a bool or boolean array, but that it is the result of `intersects` is IMO just duplicating what is already said above.

(thought the same about the `intersection` et al docstrings)